src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1293,8 +1293,8 @@ internal static bool PathIsUnc(string path)
                 return false;
             }
 
-            // handle special cases like \\wsl$\ubuntu which isn't a UNC path, but we can say it is so the filesystemprovider can use it
-            if (path.StartsWith(WslRootPath, StringComparison.OrdinalIgnoreCase))
+            // handle special cases like \\wsl$\ubuntu and \\?\ which aren't a UNC path, but we can say it is so the filesystemprovider can use it
+            if (path.StartsWith(WslRootPath, StringComparison.OrdinalIgnoreCase) || path.StartsWith("\\\\?\\"))
```

Comment:
`internal static bool PathIsNetworkPath(string path)` now return true for the volume path. Is it ok?

Reply:
I'll review all the uses of that to make sure

Reply:
This will cause some problems as the usage expects it to only be a network path.  My plan is to add a paraneter to `PathIsUnc(bool networkOnly = false)` that can be used in a single wrapper function so that only UNC returns true and not WSL or `\\?\`